js/ui/drawer/ui.drawer.js

```diff
@@ -414,6 +414,28 @@ const Drawer = Widget.inherit({
         this.$element().toggleClass(OPENED_STATE_CLASS, opened);
     },
 
+    _refreshPanel() {
+        if(this._overlay) {
+            this._overlay && this._overlay._dispose();
+
+            delete this._overlay;
+            delete this._$panel;
+            this._$panel = $("<div>").addClass(DRAWER_PANEL_CONTENT_CLASS);
+            this._$wrapper.prepend(this._$panel);
+        }
+
+        this._$panel.empty();
+
+        this._strategy.renderPanel(this._getTemplate(this.option("template")));
+    },
+
+    _setInitialPosition() {
+        $(this.content()).css("left", 0);
```

Comment:
Can we set to variables `this.content()` and `this.viewContent()`? They are used several times here.